### PR TITLE
Add pages for full websites

### PR DESCRIPTION
**Pull request by danielufeli**
- Add Home Page
- Add Quote Page
- Refactor Calculator Component
- Add NavBar with Routers

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+          <Route
+            path="/calculator"
+            component={Calculator}
+          />
+          <Route exact path="/" component={Home} />
+          <Route path="/qoute" component={Quote} />
+        </Switch>
       </>
     );
   }
```
Comment (andersonlebon) [inline src/App.js:24]: - [x] KIndly make sure at when the application starts the Home component should be rendered as the root component. Actually, when your application starts I am getting only the navigation bar :point_down:

image

- [ ] You can solve it by removing the exact key world on the Home route.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -20,10 +20,11 @@ const Calculator = () => {
...
         <div className="result">
-          { total || '' }
-          { operation || '' }
-          { next || '' }
+          {total || ''}
+          {operation || ''}
+          {next || ''}
         </div>
         <div className="buttons">
           <Buttons onBtnClick={onBtnClick} />
```
Comment (andersonlebon) [inline src/components/Calculator.js]: - [x] Kindly make sure that you follow the wireframe provided in the calculator route. You can do this by reducing the width of the calculator and aligning it to the right.  

image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @danielufeli 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @danielufeli 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**